src/apolo_app_types/protocols/jupyter.py

```diff
@@ -58,8 +58,10 @@ class Networking(AbstractAppFieldType):
     )
     http_auth: bool = Field(
         default=True,
-        description="Whether to use HTTP authentication.",
-        title="HTTP Authentication",
+        json_schema_extra=SchemaExtraMetadata(
+            description="Whether to use HTTP authentication.",
```

Comment:
nit, a rule-of-thumb we are trying to follow is to start the description with a verb. E.g.

```suggestion
            description="Enable platform provided HTTP authentication.",
```